frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -264,15 +265,22 @@ export const supportLogic = kea<supportLogicType>([
             }
 
             try {
+                const zendeskRequestBody = JSON.stringify(payload, undefined, 4)
                 const response = await fetch('https://posthoghelp.zendesk.com/api/v2/requests.json', {
                     method: 'POST',
-                    body: JSON.stringify(payload, undefined, 4),
+                    body: zendeskRequestBody,
                     headers: { 'Content-Type': 'application/json' },
                 })
                 if (!response.ok) {
                     const error = new Error(`There was an error creating the support ticket with zendesk.`)
+                    const extra: Record<string, any> = { zendeskBody: zendeskRequestBody }
+                    Object.entries(payload).forEach(([key, value]) => {
+                        extra[`payload_${key}`] = value
+                    })
+                    extra['response_status'] = response.status
+                    extra['response_body'] = await response.text()
```

Comment:
There's a proper place to set request/response metadata, see [request](https://develop.sentry.dev/sdk/event-payloads/request/) and [response](https://develop.sentry.dev/sdk/event-payloads/contexts/#response-context) docs.
So the metadata is properly formatted and highlighted.

Reply:
Another option is to use this integration https://docs.sentry.io/platforms/javascript/configuration/integrations/httpclient/ which captures failed requests automatically with all its metadata (request and response).

Reply:
I didn't test the TS bits bit the data structure is correct https://github.com/PostHog/posthog/pull/20434/commits/a40100f428c65fed5ae0638f5a0f84c295903a3c
PS: I added [this](https://github.com/getsentry/rfcs/blob/main/text/0022-response-context.md) :D